conda_libmamba_solver/solver.py

```diff
@@ -618,7 +621,18 @@ def _maybe_raise_for_problems(
         return unsatisfiable
 
     def _prepare_problems_message(self):
-        return f"{self.solver.problems_to_str()}\n{self.solver.explain_problems()}"
+        legacy_errors = self.solver.problems_to_str()
+        if "unsupported request" in legacy_errors:
+            # This error makes 'explain_problems()' crash. Anticipate.
+            log.info("Failed to explain problems. Unsupported request")
+            return legacy_errors
+        try:
+            explained_errors = self.solver.explain_problems()
+        except Exception as exc:
+            log.info("Failed to explain problems", exc_info=exc)
```

Comment:
Would it be better here to use `log.error` or `log.warning`?

Reply:
This will pollute the stdout with no real value to the user. If they miss the explanation, they can enable verbosity to see why. We shouldn't ever reach this part though so, yea, let's do it. I'll go for warning.